### PR TITLE
[fix] make tooltip body persistent on hover

### PR DESCRIPTION
**Pull request by MisterSquishy**
includes the tooltip content inside the hover target, allowing users to hover the displayed values and copy/paste:

BEFORE
2023-08-07 at 21 13

AFTER
2023-08-07 at 21 11

### CODE REVIEW

File: v2/components/tooltip/tooltip.stories.tsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ export default {
 
 export const Primary = (args: any) => {
     return (
-        <Tooltip {...args} content={<Text>{args.content}</Text>}>
-            <Text>Hover over me!</Text>
-        </Tooltip>
+        <div style={{padding:'1em', width: 'fit-content'}}>
```
Comment (MisterSquishy) [inline v2/components/tooltip/tooltip.stories.tsx:12]: just some lil cosmetic fixes to the storybook component

File: v2/components/tooltip/tooltip.tsx [hunk 2]
```diff
@@ -33,11 +33,11 @@ export const useHover = (): [React.MutableRefObject<any>, boolean] => {
 export const Tooltip = (props: {content: React.ReactNode | string; inverted?: boolean} & React.PropsWithRef<any>) => {
     const [tooltip, showTooltip] = useHover();
     return (
-        <div style={{position: 'relative', minWidth: 0}}>
+        <div ref={tooltip} style={{position: 'relative', minWidth: 0}}>
```
Comment (MisterSquishy) [inline v2/components/tooltip/tooltip.tsx:36]: scooted the ref up to the parent, which contains both the tooltip content and the trigger element

**Review by agilgur5 — APPROVED** [review]
Great usability change!
The GIF before/after and in-line comments are helpful too 🙂 

Thanks for the contribution!

**Comment by agilgur5** [comment]
cc @rbreeze @alexmt for review

Also is the @argoproj/argo-ui-maintainers team private? I can't seem to tag it

**Review by alexmt — APPROVED** [review]

**Comment by alexmt** [comment]
Thank you for attaching the recording!

**Pull request merged**